Cython/Compiler/ExprNodes.py

```diff
@@ -6976,7 +6976,11 @@ def analyse_as_ordinary_attribute_node(self, env, target):
         self.obj = self.obj.analyse_types(env)
         self.analyse_attribute(env)
         if self.entry and self.entry.is_cmethod and not self.is_called:
-#            error(self.pos, "C method can only be called")
+            if getattr(self.entry, 'is_cgetter', False):
```

Comment:
This should probably say `self.entry.type` – which suggests that this unused code is at least not tested.
Also, instead of using `getattr()` on a type, we normally set a class attribute somewhere in the type class hierarchy (or the `Entry`, or the `Node` hierarchy, …) and then override it with an instance attribute where necessary.

Reply:
While the code path is hit, it is always False since `is_cgetter` lives on `self.entry.type`. It is cruft left over from a previous iteration of the code. Reverting.